CHANGELOG.md

```diff
@@ -132,7 +56,7 @@ Some additional unsealing improvements are:
 - github.com/filecoin-project/go-state-types (v0.11.1 -> v0.12.1):
   - feat: add cbor gen for verifreg.AllocationRequest(s) (#201) ([filecoin-project/go-state-types#201](https://github.com/filecoin-project/go-state-types/pull/201))
 
-## Fixes Since RC1
+## Fixes Since 1.23.4-rc1
```

Comment:
You could consider dropping this entirely, and treat this as a "fresh" RC? I don't really care.